package.json

```diff
@@ -19,14 +19,25 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-link-records.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-link-records/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-link-records#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@angular/router": "^4.3.6",
+    "@skyux/core": "^3.1.0",
+    "@skyux/forms": "^3.0.0-rc.0",
+    "@skyux/i18n": "^3.0.1",
+    "@skyux/indicators": "^3.0.0-rc.0",
+    "rxjs": "^5.4.3"
+  },
+  "dependencies": {
+    "microedge-rxstate": "2.0.2"
```

Comment:
I spoke with Paul about putting microedge as a dependency and he said it should be a peer since it will have a shared state with other modules. Sorry for not catching that earlier (you'll need to add microedge as a dependency of skyux2, in that pull request).

Reply:
done